cli/config.go

```diff
@@ -69,6 +70,7 @@ func registerFlags(c *Config, f *flag.FlagSet) {
 	f.StringVar(&c.IntermediatesFile, "intermediates", "", "intermediate certs")
 	f.StringVar(&c.CABundleFile, "ca-bundle", "", "path to root certificate store")
 	f.StringVar(&c.IntBundleFile, "int-bundle", "", "path to intermediate certificate store")
+	f.StringVar(&c.OutFile, "out-file", "", "format for the exported file")
```

Comment:
I think this should be something like "Format", "out-file" suggests this is a filename.